lrlog/tests/lrlog_test.go

```diff
@@ -0,0 +1,128 @@
+package tests
```

Comment:
file?

Reply:
This logging package is used by heartbeat package